src/libs/shared-components/WhyChooseHorizonCard/WhyChooseHorizonCard.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Card } from '@shared-components';
+import styles from './WhyChooseHorizonCard.module.scss';
+
+export interface WhyChooseHorizonCardProps {
+  title: string;
+  listItems: string[];
+  style?: React.CSSProperties;
+}
+
+const WhyChooseHorizonCard: React.FC<WhyChooseHorizonCardProps> = ({
```

Comment:
Can we please raname this card to make it more generic? as this would be usable on other screens too.